docs/02_database/01_fields-tables-views/01_fields/01_fields-basics.md

```diff
@@ -84,6 +84,18 @@ field(name = "DESCRIPTION", type = STRING, maxSize = dbMaxSize(target = 9000))
 ```
 If the underlying database is Postgres, this sets the field's `maxSize` to 9000, because Postgres can support up to 65535. But if you are using MS SQL, then the `maxSize` is set to 8000, the maximum supported size for MS SQL.
 
+## Marking fields as sensitive
+
+You can mark fields as sensitive using `sensitive` keyword as shown in below example, which will mask sensitive field values.
+By default, following framework fields are declared as sensitive: PASSWORD, REFRESH_TOKEN and SESSION_AUTH_TOKEN
+
+```kotlin
+field(name = "PASSWORD", type = STRING, sensitive = true)
+field(name = "SESSION_AUTH_TOKEN", type = STRING, sensitive = true)
+```
+To display sensitive fields in your logs for debugging purpose, you can utilize system property called as `DisplaySensitiveFields`, if set to true all sensitive fields are displayed as plain text else they will be masked, 
```

Comment:
@ShrutiTalewad the sensitive boolean flag has been available for a long time and it always worked for generated entities. However, DisplaySensitiveFields only applies for GenesisSet and DbRecord objects, and it has a hardcoded set of fields that will be taken into account:
`        private val SENSITIVE_FIELDS: Set<String> = setOf("PASSWORD", "REFRESH_TOKEN", "SESSION_AUTH_TOKEN")
`
I think the information in this PR is misleading, because GenesisSet and DbRecord will ignore sensitive fields that are not PASSWORD, REFRESH_TOKEN and/or SESSION_AUTH_TOKEN.

Also, I have just noticed that the field written to database tables is called REFRESH_TOKEN, but the value we expose to the frontend is called REFRESH_AUTH_TOKEN. We should add REFRESH_AUTH_TOKEN to the GenesisSet set hardcoded list of sensitive fields for now to make sure we are also hiding that value.

Reply:
@ShrutiTalewad I'll rewrite this to reflect this information, ok?

Reply:
Thanks! @jacinpoz 
Also it would be good to rename this DisplaySensitiveFields property to something which says DisplayDefaultSensitiveFields as it is not a flag to display all sensitive fields instead it checks for only default ones

Reply:
we should rename that propety for both 6.6.x version and 6.7.x version before the docs as released, as I am not sure anyone is using the property just yet.

Reply:
@wjhendry as there are some changes need to be made its better if I do it, thank you!
and do we know which version of docs does the latest changes on DisplaySensitiveFields need to be part of?

Reply:
DisplaySensitiveFields is for next only - so just the document you have changed.
I'll wait for your rewrite. Thanks.